package.json

```
@@ -4,64 +4,50 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
-    "build": "preact build --no-prerender",
-    "serve": "preact build --no-prerender && preact serve",
+    "build": "preact build",
+    "serve": "sirv build --port 8080 --cors --single",
     "dev": "preact watch",
     "lint": "eslint src",
-    "test": "jest ./tests",
-    "deploy": "gulp deploy",
-    "license": "license-checker --csv --production > licenses.csv"
+    "test": "jest"
```

I think we still want the gulp dependencies for the deploy task.
```suggestion
    "deploy": "gulp deploy",
    "test": "jest"
```